packages/web-app-files/src/components/AppBar/CreateSpace.vue

```diff
@@ -33,6 +33,12 @@ export default defineComponent({
     ...mapMutations('runtime/spaces', ['UPSERT_SPACE']),
     ...mapMutations('Files', ['UPSERT_RESOURCE', 'UPDATE_RESOURCE_FIELD']),
 
+    onNewProjectButtonClick() {
+      window.open(
```

Comment:
Note for whoever ends up working on this to add an `if-else` containing the new functionality from https://github.com/owncloud/web/pull/9383